resources/definitions/tenlog_printer.def.json

```diff
@@ -0,0 +1,295 @@
+{
+    "name": "tenlog_printer",
+    "version": 2,
+    "inherits": "fdmprinter",
+    "metadata": { 
+	"visible": false,
+        "author": "Tenlog",
+        "manufacturer": "Tenlog",
+        "weight": 4,
+
+        "has_variants": false,
+        "variants_name": "Nozzle Size",
+        "preferred_variant_name": "0.4mm Nozzle",
```

Comment:
Since the printer has no variants, the `variants_name` and `preferred_variant_name` should not be mentioned.
```suggestion
```